src/spikeinterface/postprocessing/template_metrics.py

```diff
@@ -132,14 +197,31 @@ def get_extension_function():
 WaveformExtractor.register_extension(TemplateMetricsCalculator)
 
 
+_default_function_kwargs = dict(
+    recovery_window_ms=0.7,
+    peak_relative_threshold=0.2,
+    peak_width_ms=0.1,
+    depth_direction="y",
+    min_channels_for_velocity=5,
+    min_r2_velocity=0.5,
+    exp_peak_function="ptp",
+    min_r2_exp_decay=0.5,
+    spread_threshold=0.2,
+    spread_smooth_um=20,
+    column_range=None,
+)
+
+
 def compute_template_metrics(
     waveform_extractor,
-    load_if_exists=False,
-    metric_names=None,
-    peak_sign="neg",
-    upsampling_factor=10,
-    sparsity=None,
-    window_slope_ms=0.7,
+    load_if_exists: bool = False,
+    metric_names: Optional[list[str]] = None,
+    peak_sign: Optional[str] = "neg",
+    upsampling_factor: int = 10,
+    sparsity: Optional[ChannelSparsity] = None,
+    include_multi_channel_metrics: bool = False,
+    metrics_kwargs: dict = None,
+    debug_plots: bool = False,
```

Comment:
I am OK to have debug_plot inside the code at the tricky places but I would avoid this in the signauture

Reply:
Ok, I can remove it and set it as a flag on top of the file